governance/writing-style.md

```diff
@@ -0,0 +1,66 @@
+# Writing style
+
+Consistency creates clarity in communication.
+
+If you find yourself correcting for consistency, please propose additional style
+guidelines via pull request to this document. Feel free to add references to
+good sources for content guidelines at the bottom of this guide.
+
+## Common terms
+
+* When referring to users and use cases, ensure consistency with use cases
+* See [CNCF Style Guide][cncf-style] for common terms. Note that the following terms are not hyphenated and all lower case, except for capitalizing the first letter when at the beginning of a sentence:
+  * open source
+  * cloud native
+
+## Additional formatting
+
+* Headlines, page titles, subheads and similar content should follow sentence
+  case, and should not include a trailing colon.
+* Paragraphs do not start with leading indent.
+* Wrap lines at 80 characters, except where it would break a link. No need to
+  reformat the whole paragraph to make it perfect -- fewer diffs are easier
+  for reviewers.
+
+## File & directory naming conventions
+
+* Every directory should have a README.md with useful introductory text.
+* All other file and directory names should be all lower case with dashes to
+  separate words.
+
+## Pull Request Quality Checks
+
+As part of every pull request following quality checks are being executed to ensure that submitted content is compliant with existing best practices for spelling, document formatting and link formats:
+
+* [markdownlint-cli2](https://github.com/DavidAnson/markdownlint-cli2) tool is used to verify correct Markdown file formatting;
+* [cSpell](http://cspell.org) is used to verify that content is compliant with established spelling rules;
+* [markdown-link-check](https://github.com/tcort/markdown-link-check) is used to verify that all the links are valid and properly constructed;
+
+In case any of the modified files in the PR violate any of the checks a "Quality Checks" build will fail.
+
+### Running Quality Checks locally
+
+We encourage everyone to execute quality checks locally prior to creating a pull request to ensure that you can fix as many errors as possible before going through a code review process. To run the quality checks locally you can run ```npm run qa:dev``` command from the root of the tag-env-sustainability repository - this command will trigger a [qa-checks-local.sh](./qa-checks-local.sh) script that will perform all the validations mentioned above on the files that you've modified. It will also install the tools mentioned above if those don't exist locally. If any issues will be identified by the script those will be written to the terminal session where the command has been triggered so that you can easily go through and follow up the detected errors accordingly.
```

Comment:
```suggestion
We encourage everyone to execute quality checks locally prior to creating a pull request to ensure that you can fix as many errors as possible before going through a code review process. To run the quality checks locally you can run ```npm run qa:dev``` command from the root of the tag-env-sustainability repository - this command will trigger a [qa-checks-local.sh](../qa-checks-local.sh) script that will perform all the validations mentioned above on the files that you've modified. It will also install the tools mentioned above if those don't exist locally. If any issues will be identified by the script those will be written to the terminal session where the command has been triggered so that you can easily go through and follow up the detected errors accordingly.
```